server/src/main/java/io/druid/server/ClientQuerySegmentWalker.java

```diff
@@ -84,25 +82,13 @@ public ClientQuerySegmentWalker(
     final FinalizeResultsQueryRunner<T> baseRunner = new FinalizeResultsQueryRunner<T>(
         toolChest.postMergeQueryDecoration(
             toolChest.mergeResults(
-                new MetricsEmittingQueryRunner<T>(
```

Comment:
I think notifying the users makes sense, release-notes for 0.7.1 would be the best place. Is there a way for me to update those?